locksmith/src/graphql/datasource/keysByQuery.ts

```diff
@@ -4,6 +4,7 @@ import gql from 'graphql-tag'
 import { DocumentNode } from 'graphql'
 import { getValidNumber } from '../../utils/normalizer'
 
+// todo: replace with subgraphService
```

Comment:
Oh yes please open an issue !

Reply:
Opened and assigned to myself 
https://github.com/unlock-protocol/unlock/issues/11337